cmd/crossplane/core/core.go

```diff
@@ -147,6 +146,10 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 		//nolint:revive // This is long and easier to read with punctuation.
 		return errors.New("Crossplane removed support for external secret stores. The --enable-external-secret-stores flag will be removed in a future release.")
 	}
+	if c.Registry != "" {
+		//nolint:revive // This is long and easier to read with punctuation.
+		return errors.New("Crossplane removed support for the --registry flag, as a default registry is not supported. Please fully qualify images by explicitly providing a registry.")
```

Comment:
can you also update the v2 docs to list this as a breaking change too? https://docs.crossplane.io/v2.0-preview/whats-new/#backward-compatibility